hathor/transaction/storage/transaction_storage.py

```diff
@@ -756,6 +756,47 @@ def is_rocksdb_storage(self) -> bool:
         """
         return False
 
+    def iter_mempool_tips_from_tx_tips(self) -> Iterator[Transaction]:
```

Comment:
Should we add a test that enables both indexes and compare that they both return equivalent iterators?

Reply:
That would be great.

Reply:
Have you done this?

Reply:
I added `test_compare_mempool_implementations` to `tests/simulation/test_simulator.py`, which basically checks the two implementations during the node's simulated workloads.

Reply:
Perfect! Great job! Thanks!